.github/workflows/tests.yml

```diff
@@ -201,11 +201,13 @@ jobs:
 
     - name: install harbor-operator
       run: |
+        make kustomize
+
         cd manifests/cluster
         kustomize edit add secret github-token --disableNameSuffixHash --from-literal=GITHUB_TOKEN=${{ secrets.GITHUB_TOKEN }}
         kustomize edit add patch --path patch/github-token.yaml
         kustomize edit set image goharbor/harbor-operator=${dockerImage}
-        kustomize build | kubectl create -f -
+        ../../bin/kustomize build --reorder legacy | kubectl create -f -
```

Comment:
Necessary - we previously used `kustomize` shipped with ubuntu `latest` it seems. That caused us to hit deprecations in the current setup and therefore things were applied in the wrong order. Using kustomize from the make file fixed it.

Reply:
Why do we need **--reorder legacy**?

Reply:
It's used consistently through the makefile in this repo. The issues I encountered were definitely because of kustomize, so I aligned it with the rest of the repo ordering.

See here for example: https://github.com/goharbor/harbor-operator/blob/9c19b151ff71f66f9079236e4dc4ce3d4a1e02c3/Makefile#L444

I will file a follow up issue to remove it everywhere.

Reply:
Issue: https://github.com/goharbor/harbor-operator/issues/1021